575.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @param {number[]} candyType
+ * @return {number}
+ */
+var distributeCandies = function (candyType) {
+  const resultOfCandy = candyType.length / 2;
+  const sorted = [...new Set(candyType)].sort((a, b) => a - b);
+
+  if (resultOfCandy < sorted.length) {
+    return resultOfCandy;
+  } else {
+    return sorted.length;
+  }
```

Comment:
You can also use _Conditional Operator_ instead of the if-else statement, which reduces the amount of code.
 _Conditional Operator_ is a useful operator, so check out [the documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator)
```suggestion
  return resultOfCandy < sorted.length ? resultOfCandy : sorted.length;
```